app/models/edition.rb

```diff
@@ -266,7 +266,7 @@ def has_video?
   end
 
   def safe_to_preview?
-    true
+    true unless archived?
```

Comment:
This will return nil, rather than false for archived editions. It might be nicer to write it as `not archived?`?